Jenkinsfile

```diff
@@ -15,13 +15,22 @@ pipeline {
         stage("Build & Test") {
             agent {            
                 dockerfile {
+<<<<<<< HEAD
```

Comment:
Looks like a bad merge here for some reason. I'd just take the files from master